### DISCUSSION TITLE
Alternatives to FBA Manage Inventory Health Report on SP API

### DISCUSSION THREAD

**Discussion by Saurabhomgupta** [amzn/selling-partner-api-models#3520] [General] [answered]
Hi Team,

Is there an alternative available in SP API that can be used to replace the FBA Manage Inventory Health report with the similar metrics including Rank, Inventory Aging. 

Thanks
Saurabh

**Comment by sungolivia** [comment, marked as answer]
Hey @Saurabhomgupta have you seen this [page](https://developer-docs.amazon.com/sp-api/docs/report-type-values#fulfillment-by-amazon-fba-reports) before? 
![Screenshot 2023-02-21 at 8 58 48 AM](https://user-images.githubusercontent.com/7606170/220410700-f446b39d-4521-4227-bc01-e26532b47285.png)

**Comment by Saurabhomgupta** [comment]
Ahh.. I never noticed the name change.. Thanks for this. @sungolivia
